src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeToFile(report, toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder data = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                data.append(line)
+                        .append(System.lineSeparator());
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return data.toString();
+    }
+
+    private String createReport(String dataFromFile) {
+        int supplyTotal = 0;
+        int buyTotal = 0;
+        StringBuilder reportBuilder = new StringBuilder();
+
+        String[] lines = dataFromFile.split(System.lineSeparator());
+        for (String line : lines) {
+            String[] parts = line.split(",");
+            if (parts.length == 2) {
+                String operationType = parts[0];
```

Comment:
Indexes can be stored as constants with descriptive name.

Reply:
So i should add it like that?:

public class WorkWithFile {
private static final int OPERATION_TYPE_INDEX = 0;
private static final int AMOUNT_INDEX = 1;

rest of the code...

can't set those constants inside the method body, or just before it, im obligated to declare them in this place, right?